src/main/scala/com/amazon/deequ/examples/PengExample.scala

```diff
@@ -0,0 +1,60 @@
+package com.amazon.deequ.examples
```

Comment:
Please rename this file and give it a license header

Reply:
Actually I wrote this example for testing how much time it could take to run deequ checks on different scales of datasets. It is irrelevant to the KLL Sketch and I will remove this file.